tests/test_markers.py

```diff
@@ -2,12 +2,11 @@
 # 2.0, and the BSD License. See the LICENSE file in the root of this repository
 # for complete details.
 
-import collections
 import itertools
 import os
 import platform
 import sys
-from typing import cast
+from typing import NamedTuple, cast
```

Comment:
Importing objects directly from `typing` is just to make type hints easier to read. Since `NamedTuple` isn't used in type hints it would be better to inherit `typing` itelf.

```suggestion
import typing
from typing import cast
```

Reply:
> What's the point of the change if there's no change in typing?

Yes, good question. I was thinking of adding more typing, but then we just need to test this function:
```python
def format_full_version(info: sys._version_info) -> str:
    version = f"{info.major}.{info.minor}.{info.micro}"
    kind = info.releaselevel
```
We need something compatible with type annotation `sys._version_info` which is not really documented as far as I can see. I think this won't get us anywhere, especially for test code. See [What is the type of "sys.version_info"?](https://stackoverflow.com/questions/57862106/what-is-the-type-of-sys-version-info)

Reply:
This looks like a case where ruff/flake8-pyi rule [`PYI024`](https://docs.astral.sh/ruff/rules/collections-named-tuple/) won't improve things.

Reply:
> This looks like a case where ruff/flake8-pyi rule [`PYI024`](https://docs.astral.sh/ruff/rules/collections-named-tuple/) won't improve things.

That sounds like we should just close this PR then.